core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -582,6 +588,26 @@ private[spark] class ExecutorAllocationManager(
     }
   }
 
+  // Metric source for ExecutorAllocationManager to expose the its internal executor allocation
```

Comment:
Nit: "the" and "its" are redundant.  Also, this should be a block-style comment.